### PR TITLE
Secure Your Gaming Company

### PR DESCRIPTION
**Pull request by dan-auth0**

### CODE REVIEW

File: _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown [hunk 1]
```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: "Secure Your Gaming Company With Auth0's User Fraud Score and MinFraud"
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown]: Initial title too long for SERPS.

Suggested: "Secure Your Gaming Company With Auth0's Fraud Protection"

File: _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown [hunk 2]
```diff
@@ -0,0 +1,187 @@
+---
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown]: General: if accepting new, suggested title changes, let's change filename/url to reflect this, please.

File: _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+layout: post
+title: "Secure Your Gaming Company With Auth0's User Fraud Score and MinFraud"
+description: "Rapid growth across mobile, console, and PC platforms can lead to disaster without proper protections."
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown]: Suggested description: "Rapid gaming industry growth across mobile, console, and PC platforms can lead to disaster without proper security and anti-fraud protections."

File: _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  - anomaly
+  - fraud
+  - fraud-score
+  - minfraud
+  - rules
+  - authentication
+  - socure
+  - ciam
+related:
+  -
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown]: Let's definitely include this one as related post: `2018-02-16-the-6-billion-reason-your-business-needs-advanced-fraud-protection` and this one: `2017-10-27-where-video-game-authentication-falls-short-and-how-gamers-can-stay-safe`

File: _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+  - minfraud
+  - rules
+  - authentication
+  - socure
+  - ciam
+related:
+  - 
+---
+
+In April 2018, Epic Games’ _Fortnite_ generated [$296 million](https://techcrunch.com/2018/05/25/fortnite-had-a-296-million-april/)—more than double what it generated in February. Growth was rapid across mobile, console, and PC platforms.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown]: Included Fortnite in anchor, remove italics.

`[Fortnite generated $296 million](https://techcrunch.com/2018/05/25/fortnite-had-a-296-million-april/)`

File: _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+At the same time as users peaked in February 2018 with [3,400,000 simultaneous logins, mysterious charges](https://kotaku.com/whats-really-going-on-with-all-those-hacked-fortnite-ac-1823965781) began to appear in users' accounts, ranging from $99.99 to $149.99. Hackers made off with one-off charges and often racked up multiple thefts from single users. [One player even struggled to pay his rent](https://www.reddit.com/r/FORTnITE/comments/841zdn/account_hacked_epic_please_help_thats_my_rent/) as a result of the total financial damage. Thieves exposed critical data via multiple log-in attempts, posted sign-in codes, and offered upgrades at steep discounts.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown]: Suggested to move anchor around a bit: `[mysterious charges began to appear in users' accounts](https://kotaku.com/whats-really-going-on-with-all-those-hacked-fortnite-ac-1823965781)`

File: _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+The example isn't isolated. Annually, [cyber thieves make off with over 40 percent](https://www.panopticonlabs.com/our-technology/) of video-game publishers' in-game revenue. As the global industry [sales top $100 billion](https://seekingalpha.com/article/4184342-video-games-taking-will-esports-become-larger-sports), more criminals are taking notice, and more companies are at risk.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown]: `As the global industry [sales top $100 billion](https://seekingalpha.com/article/4184342-video-games-taking-will-esports-become-larger-sports), more criminals...`

Suggested: `As the global [gaming industry's sales top $100 billion](https://seekingalpha.com/article/4184342-video-games-taking-will-esports-become-larger-sports)...`

File: _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+
+While Auth0 has several Rules, all of which developers can use to customize and [extend Auth0's current suite of capabilities](https://auth0.com/b2c-customer-identity-management) to understand and protect end users, we've highlighted two particular ones below that growing video game companies can implement for better security as they scale.
+
+These can be used separately or chained together for modular coding.
+
+### User Fraud Score
+
+The User Fraud score is particularly helpful in confirming that users are who they say they are and in quickly blocking those who are behaving inappropriately. 
+
+[Auth0 has partnered with Socure](https://www.socure.com/) to develop a user fraud score for each account that authenticates, based on their email address and IP address.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown]: Just include Socure: `Auth0 has partnered with [Socure](https://www.socure.com/)...`

File: _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+
+* suspicious pairings of user location with address and IIN data;
+* risk associated with the IP address; and
+* reputation checks for street addresses, email addresses, phone numbers, and devices.
+
+In cases like _Fortnite_, minFraud could have helped the team block transactions that didn't meet standards for the above criteria, stopping theft from spreading throughout its network.
+
+## Use Rules As Part of a Larger CIAM Strategy
+
+Adding individual Rules that pertain to certain users on your platform is a step in the right direction. Setting up a comprehensive strategy for more deeply understanding all of your users is the next. Either [building or outsourcing](https://auth0.com/b2c-customer-identity-management) customer identity and access solutions can bring a new dimension to your insights.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-secure-your-gaming-company-with-auth0's-user-fraud-score-and-minfraud.markdown]: "customer identity and access **management** solutions"

**Comment by dan-auth0** [comment]
Done. Thank you.

**Pull request merged**